ouroboros-consensus-test/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -785,11 +799,13 @@ withTestMempool setup@TestSetup {..} prop =
 
       -- Set up the LedgerInterface
       varCurrentLedgerState <- uncheckedNewTVarM testLedgerState
-      trivialLedgerBackingStore <- trivialBackingStore polyEmptyLedgerTables
       let ledgerInterface = LedgerInterface
-            { getCurrentLedgerAndChangelog = (, MempoolChangelog Origin polyEmptyLedgerTables) <$> readTVar varCurrentLedgerState
-            , getBackingStore  = pure $ LedgerBackingStore trivialLedgerBackingStore
-            , withReadLock     = id
+            { getCurrentLedgerState = forgetLedgerTables <$> readTVar varCurrentLedgerState
+            , getLedgerTablesAtFor = \pt _ -> do
```

Comment:
Are we returning a state that contains more than the requested keys? Why no returning just those keys?

Reply:
Changed, good point